Holopin (Independent Publisher) v1.1

troystaylor opened the pull request:
Updating for #Hacktoberfest
CC: @Holopin

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

troystaylor committed:
Add files via upload

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Add files via upload

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Add files via upload

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Merge branch 'microsoft:dev' into dev

troystaylor committed:
Add files via upload

troystaylor committed:
Merge branch 'microsoft:dev' into Holopin

troystaylor committed:
Update to v1.1

vmanoharas commented:
Hi @troystaylor, 

Kindly upload the connector package zip file. 

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission-ip

troystaylor committed:
Update apiDefinition.swagger.json

troystaylor committed:
Enhance Holopin readme with detailed operations

Updated the readme to include more detailed information about Holopin, its operations, and known issues.

troystaylor committed:
Delete independent-publisher-connectors/Holopin/Holopin directory

troystaylor committed:
Add files via upload

troystaylor commented:
@vmanoharas Zip uploaded.

vmanoharas commented:
certify-connector

certificationteam commented:
Congratulations — your certification request has been approved. Your connector will be included in the upcoming deployment schedule for production release and the supporting documentation will be published. This end-to-end process typically takes 3–4 weeks.

Many Thanks,
Copilot Connector Certification Team

vmanoharas reviewed (approved):
Dear Partner

I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

vmanoharas commented:
Hello Partner,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, 
Thank you very much for working with us.
